src/structs/index.md

```diff
@@ -5,26 +5,26 @@ languages. As long as you tell the Rust compiler to lay out structs "how C does
 it" (with `#[repr(C)]`) and have the right type declarations, everything *Just 
 Works™*. It's all bytes at the end of the day, anyway.
 
-Remember how I mentioned earlier that there's an easier way to compile and link
-everything? Well there is, it's called [gcc-rs][gcc-rs]. All you need to do is 
+Remember that there's an easier way to compile and link everything?
```

Comment:
How about, "Remember when we said there's an easier way to compile and link everything?"? The current sentence sounds a bit like it's confused whether to be a sentence or a statement.